src/main/java/com/ays/backend/user/model/EStatus.java

```diff
@@ -0,0 +1,9 @@
+package com.ays.backend.user.model;
+
+public enum EStatus {
+    WAITING,
+    VERIFIED,
```

Comment:
Please add values to these enums, as you'll have indices instead, when you want to get the value of it.